examples/dhf5-custom-hook/build.gradle

```diff
@@ -1,19 +1,8 @@
-buildscript {
```

Comment:
I used this project for ad hoc testing and updated it to reference 5.0.0, now that that's available.